cllc-public-app/ClientApp/src/app/components/applications/permanent-change-to-a-licensee/permanent-change-to-a-licensee.component.ts

```diff
@@ -769,4 +770,36 @@ export class PermanentChangeToALicenseeComponent extends FormBase implements OnI
   ngOnDestroy(): void {
     this._debouncedSave.cancel();
   }
+
+  private scrollToFirstInvalidControl() {
+    // Grab the first invalid control
+    var firstInvalidControl: HTMLElement = document.querySelector('.form-wrapper .ng-invalid');
+
+    if (firstInvalidControl) {
```

Comment:
nitpick: Id suggest inverting this `if`, to avoid needing to nest everything after it.

```
if (!firstInvalidControl) {
  return;
}

const panel = ...
...
```